src/code-util.jl

```diff
@@ -189,43 +199,25 @@ function extract_problems(results)
     rel_code_key = "/:rel/:catalog/:diagnostic/:code/Int64/String"
     rel_line_key = "/:rel/:catalog/:diagnostic/:range/:start/:line/Int64/Int64/Int64"
     rel_severity_key = "/:rel/:catalog/:diagnostic/:severity/Int64/String"
+    rel_message_key = "/:rel/:catalog/:diagnostic/:message/Int64/String"
```

Comment:
since these are also used in testing, it might be worth pulling them out into the global namespace and making them constant (e.g. `const REL_CODE_KEY = "..."`). then they could be imported for the tests

Reply:
Oh, I hadn't considered testing! I had been thinking of them as used here alone.

Reply:
Done.